modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js

```diff
@@ -49,16 +49,14 @@ export default function DropdownMenu({
 				})}
 				trigger={
 					<ClayButton className={cssClass} {...otherProps}>
-						{icon && (
-							<span
-								className={classNames({
-									'inline-item': label,
-									'inline-item-before': label,
-								})}
-							>
+						{icon &&
```

Comment:
Why is this change needed? If there's no `label`, we should be simply adding an empty `span`... how's that problematic?

Reply:
I mean... I see it pushes the icon slightly down... but why? :D

Reply:
/cc @pat270, @eduardoallegrini, @marcoscv-work, nothing urgent, we can go with this fix, but I'm curious as to why our markup is so picky here... 🤔

Reply:
Actually, nevermind... it's just that we need `inline-item`but not `inline-item-before`, so we should simply remove tweak that to always add `inline-item`, I think.